internal/cloudapi/v2/handler.go

```diff
@@ -329,6 +329,22 @@ func (h *apiHandlers) PostCompose(ctx echo.Context) error {
 		}
 	}
 
+	if request.Customizations != nil && request.Customizations.Openscap != nil {
+		if bp.Customizations == nil {
+			bp.Customizations = &blueprint.Customizations{
+				OpenSCAP: &blueprint.OpenSCAPCustomization{
+					DataStream: request.Customizations.Openscap.Datastream,
+					ProfileID:  request.Customizations.Openscap.ProfileId,
+				},
+			}
+		} else {
+			bp.Customizations.OpenSCAP = &blueprint.OpenSCAPCustomization{
+				DataStream: request.Customizations.Openscap.Datastream,
+				ProfileID:  request.Customizations.Openscap.ProfileId,
+			}
+		}
+	}
```

Comment:
nitpick: I like the way used by other customizations above nicer. IOW, define the OSCAP customization only once and then use it in the if-else block.